angular-app/src/app/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -0,0 +1,7 @@
+<mat-dialog-content>
+  <p>{{data}}</p>
+</mat-dialog-content>
+<mat-dialog-actions>
+  <button mat-button matDialogClose="cancel" class="primary">Cancel</button>
+  <button mat-button matDialogClose="confirm" class="warn">Continue</button>
```

Comment:
I don't doubt that these classes are the right ones to use, but the names are a bit confusing for someone maintaining this code.

Reply:
I used the wrong attribute - changed to `color`, which pulls colors from the global theme